src/index.html

```diff
@@ -20,8 +21,465 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="page">
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="/src/images/logo.png"
+        class="header__logo-link"
+      >
+        <img
+          src="/src/images/logo.png"
```

Comment:
Ensure that the `src` attribute for the image points to the correct path relative to the HTML file. Currently, it points to `/src/images/logo.png`, which may not be correct if the HTML file is in the `src` directory.